app/src/androidTest/java/ch/epfl/sdp/peakar/points/ComputePOIPointsTest.java

```diff
@@ -106,69 +109,7 @@ public void computeVerticalBearing(){
         verticalBearing = samePoint.setVerticalBearing(userPoint);
         assertEquals(NaN, verticalBearing, 1);
     }
-
-    /**
-     * Test if ComputePOIPoints has computed the POIs
-     */
-    @Test
-    public void getPOIPointsTest() {
```

Comment:
How is this part tested now?

Reply:
Not yet, it will be updated when the PR which makes the computePOIPoints class observable gets merged.

Reply:
Ok understood